db-engine/src/main/java/com/splicemachine/db/impl/ast/CorrelatedPushDown.java

```diff
@@ -26,15 +26,15 @@
 package com.splicemachine.db.impl.ast;
 
 
-import com.google.common.base.Function;
+import org.spark_project.guava.base.Function;
 import com.splicemachine.db.iapi.error.StandardException;
 import com.splicemachine.db.iapi.sql.compile.Optimizable;
 import com.splicemachine.db.iapi.sql.compile.Visitable;
 import com.splicemachine.db.impl.sql.compile.*;
 import org.apache.commons.lang3.tuple.Pair;
 import org.apache.log4j.Logger;
-import org.sparkproject.guava.collect.Collections2;
-import org.sparkproject.guava.collect.Iterables;
```

Comment:
Good point.  Would you object to starting out with Spark's and then moving to our own a bit later?  I think it would just be a find and replace.